LandSandBoat/server
# Saintly Invitation ensure cleanup of smn pets

**_I affirm:_**

## What does this pull request do?
https://www.bg-wiki.com/ffxi/Windurst_Mission_6-2 has a smn mob with both an elemental that starts and an avatar that comes out with astral flow. Under certain circumstances the avatar can stay in the battlefield after the bcnm is complete, simplest way to replicate is to simply teleport out just as it appears. Because the avatars weren't listed in the bcnm table, they weren't being cleaned up after the bcnm completes:

![image](https://github.com/LandSandBoat/server/assets/131182600/b3a354b7-8e45-46d5-b206-084eca718de0)

Adding the elementals and avatars with condition zero solves the issue, also confirmed that the windy mission still completes no problem:

![image](https://github.com/LandSandBoat/server/assets/131182600/1c8df53d-f4c1-44d7-b429-6dbc15f73031)


## Steps to test these changes
Simplest way to test is to remove the `nationStatus == 1` from line 646 in `bcnm.lua` (I'm still not adept at changing mission progress variables with the interaction framework)
```
!setplayernation 2
!addmission 2 17
!zone 146
```
interact with the bcnm and choose Saintly Invitation
run `!hp 1` on Chaa_Paqa_the_Profound and he will use astral flow, immediately `!zone 146`. This should cleanup the battlefield, and with the changes in this PR that will include the Yagudo's Avatar
